lessons/basic_mc.ipynb

```diff
@@ -89,6 +89,7 @@
     "In other words, given a list of samples of $\\theta$ from $P(\\theta)$,\n",
     "* the marginalized 1D posterior for $\\theta_0$ is estimated by making a histogram of $\\theta_0$ samples\n",
     "* the marginalized 2D posterior for $\\theta_0,\\theta_1$ is estimated from a 2D histogram of $\\theta_0,\\theta_1$ samples\n",
+    "* the posterior mean of $\\theta_0$ is estimated by a simple average over the $\\theta_0$ samples\n",
```

Comment:
I don't really see the value in adding this. Going from a posterior to means and other estimates is covered earlier, and posterior means are not something I want to promote particularly.
```suggestion
```

Reply:
Fair enough. I just know that when I was learning this stuff, the posterior mean was an easier integral to relate to than the histogram bin values.

Reply:
I think I will add back a comment about computing means and percentiles from samples later.